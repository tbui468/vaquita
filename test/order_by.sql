if exists drop database sol;
create database sol;
open sol;

create table planets (
                        id int key,
                        name string, 
                        mass float, 
                        number_of_moons int,
                        surface_pressure float,
                        ring_system bool
                    );

insert into planets (
                        id,
                        name, 
                        mass, 
                        number_of_moons, 
                        surface_pressure,
                        ring_system
                    ) 

values
        (1, "Venus", 2.32, 0, 92.0, false),
        (2, "Earth", null, 1, 1.0, false),
        (3, "Mars", 4.23, 2, 0.01, false);

select * from planets order by name;
select * from planets where number_of_moons >= 1 order by number_of_moons desc;
select * from planets where number_of_moons >= 1 order by number_of_moons;
close sol;
drop database sol;
exit;
