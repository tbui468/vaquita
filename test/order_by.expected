created database sol
opened database sol
created table planets
inserted 3 record(s) into planets
id, name, mass, number_of_moons, surface_pressure, ring_system
2, Earth, null, 1, 1.000000, false
3, Mars, 4.230000, 2, 0.010000, false
1, Venus, 2.320000, 0, 92.000000, false
id, name, mass, number_of_moons, surface_pressure, ring_system
3, Mars, 4.230000, 2, 0.010000, false
2, Earth, null, 1, 1.000000, false
id, name, mass, number_of_moons, surface_pressure, ring_system
2, Earth, null, 1, 1.000000, false
3, Mars, 4.230000, 2, 0.010000, false
closed database sol
dropped database sol
